versions/v1/units.json

```
@@ -8665,6 +8665,35 @@
     "source": "qudt.org",
     "sourceReference": "https://qudt.org/vocab/unit/BBL_US-PER-DAY"
   },
+  {
+    "externalId": "volume_flow_rate:bbl_us_pet-per-hour",
```

Rule for externalId is not met
```suggestion
    "externalId": "volume_flow_rate:bbl_us_pet-per-hr",
```